CONTRIBUTING.md

```diff
@@ -159,22 +144,30 @@ Before creating your pull request, please make sure to review the `tedana` [styl
 
 ### 5. Test your changes
 
-#### Changes to code
+You can run style checks by running the following
 
-For changes to the codebase, we suggest using our development Docker container which will run all the necessary checks and tests to ensure your code is ready to be merged into `tedana`!
-(This does require that you have a local install of [Docker](https://www.docker.com/products/docker-desktop).)
-You can run all the checks with:
+```
+cd TEDANADIR
+flake8 tedana
+```
 
+and unit/integration tests by running the following:
```

Comment:
Since you give a number of options below, can we either add more explanatory text here or break up that block to reference the different calls separately ?